README.md

```diff
@@ -14,6 +14,10 @@ It works well for GitHub Action. ref: https://help.github.com/en/articles/about-
 
 **Required**. Must be in form of `github_token: ${{ secrets.github_token }}`'.
 
+### `github_ref`
```

Comment:
Can you change it to just `tag` so that all users need to specify is only tag part?